### ISSUE TITLE
Data Entry - entries from previously entered record appear

### ISSUE THREAD

**Issue by mzperry** [dancrocker/BRCWQDM#15]
There are some "remnants" of the previously submitted record on the new entry screen.  I think it's just time, weather 48, weather now and NAV.  Gave some thought as to whether this was  a bad thing especially for fields like weather where the result is likely to repeat.  Concluded that it might lead to entry errors when a default is accidentally left.  The only place it might work is date.  For everything else it's better to start with a clean slate for each new record.

**Comment by dancrocker** [comment]
Agreed - for radioButtons (single choice) I have to reset state to a value - it does not work to reset with nothing selected. For now I have reset to "Not Recorded", however this could be problematic if in reality an entry was omitted by mistake. I can add another default choice of "Nothing Selected" so that it always returns to this value. Sound ok? I cannot figure out a way to get the time to reset to 00:00... I'll keep working on that.

**Comment by mzperry** [comment]
Tricky one.  Is "nothing selected" an option on the entry page or a hidden default value?  I think it  would be more confusing to have a selection visible that you're never supposed to select.  In fact, the app should scold you if you don't select anything.

Don't lose sleep over the time not resetting.  I didn't find it to be that much of a problem.
